episodes/severity-static.Rmd

```diff
@@ -0,0 +1,640 @@
+---
+title: 'Estimate severity'
+teaching: 10
+exercises: 2
+editor_options: 
+  chunk_output_type: inline
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- How to estimate the Case Fatality Ratio (CFR)?
+
+- Why adjust a CFR with a delay distribution?
+
+- When should we estimate the CFR in an ongoing outbreak?
```

Comment:
> "When should we estimate the CFR in an ongoing outbreak?" as an Overview item - I didn't really understand what was meant here.

My intention was to express the idea of the possibility of getting an unbiased early-stage estimate, instead of a naive estimate only stable at later stages.

I found valuable also the first suggestions so rephrased the questions in https://github.com/epiverse-trace/tutorials-middle/pull/1/commits/8cee67065a59b2d822ae74700f66b2bb6120c081